rpm/0001-Remove-unsupported-csh-and-ksh.patch

```diff
@@ -0,0 +1,36 @@
+From 2ccaa3fd65388deccd45afbc7f4c5ddb33491587 Mon Sep 17 00:00:00 2001
+From: Matti Kosola <matti.kosola@jolla.com>
+Date: Tue, 26 Feb 2019 02:14:09 -0500
+Subject: [PATCH] Remove unsupported csh and ksh
+
```

Comment:
We should add comment here that this is done as having these files will automatically add dependency to the csh and ksh that we do not have and thus the ltp can't be installed.